nuxt.config.js

```diff
@@ -67,7 +67,26 @@ export default {
 
   tailwindcss: {
     config: {
-      plugins: [tailwindTypography]
+      plugins: [tailwindTypography],
+      theme: {
+        typography: {
+          DEFAULT: { // edit the prose class
+            css: {
+              'font-weight': '100',
+              p: {
+                'font-weight': '100',
+                color: '#2b32a4'
+              },
+              a: {
+                'font-weight': '300',
+                '&:hover': {
+                  color: '#a0a0a0'
+                }
+              }
+            }
+          }
+        }
+      }
```

Comment:
Does this overwrite the defaults or replace them? I would think we want to overwrite only the elements that we want to be different, and not "clear" all the other defaults.